pkg/config/default.go

```diff
@@ -544,3 +548,12 @@ func (c *Config) LogDriver() string {
 func (c *Config) MachineEnabled() bool {
 	return c.Engine.MachineEnabled
 }
+
+// RootlessNetworking returns the "kind" of networking
+// rootless containers should use
+func (c *Config) RootlessNetworking() string {
+	if len(c.Containers.RootlessNetworking) < 1 {
```

Comment:
if you do a RootlessNetworking=DefaultRootlessNetwork on  line 210, you can also remove this here
```
	if len(c.Containers.RootlessNetworking) < 1 {
                return DefaultRootlessNetwork
	}
```